.travis.yml

```diff
@@ -8,6 +8,8 @@ env:
     # Set the minimum Android API to support
     # Note: Not all of them work well with Travis
     - ANDROID_TARGET=android-21
+    # Codcov token
+    - CODECOV_TOKEN="b2a0428d-3c42-4c5e-b7c6-1563a33e76c6"
```

Comment:
Should you be committing this?

Reply:
I tried to run it without the token in the previous build  but it didn't work

Reply:
The documentation says it is not needed for public repositories, We are still setting it up

Reply:
You should set it as a (secret) environment variable in Travis, don't expose it. Now it's compromised so I'd suggest generating a new one.

Reply:
Yes sure That's not a problem, we are not sure if we want to use the service at all, so if we manage to set it up we'll update it accordingly.

Reply:
Apparently I'm not able to trigger a new build with the new Travis config, Do you know how to trigger one or do I need to close and reopen the pull request?

Reply:
A new token has been generated btw :)

Reply:
I've seen codecov getting stuck or in a bad shape sometimes.
You can trigger a new build if you either push something new, create a new pull request or manually run the codecov command you run in your Travis script **while"** having set the appropriate environment variable to your API token.